install/kubernetes/helm/istio/values-istio-gateways.yaml

```diff
@@ -55,6 +55,7 @@ gateways:
     loadBalancerIP: ""
     serviceAnnotations: {}
     type: LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be
+    externalTrafficPolicy: Local
```

Comment:
I don't think we can default to Local - it's a big change and didn't have enough testing. 

For example docs are pretty clear that it'll break (drop packets) if NodePort is used. 

/hold

Reply:
It's only added for `type: LoadBalancer` and not `NodePort`.

Reply:
And is it working on all supported platforms ? Who tested ? 

Unless there is clear evidence it is supported everywhere - please add an option and maybe off by default, we can't break stuff.

Reply:
@costinm who actually tests on all platforms?

Reply:
Until actual tests are added for all interested platforms, the general rule for every commit is to pass pre-submit.

Reply:
Pre-submit is not sufficient for a release - if you want you can make this change in master, and have this sorted out in the stabilization/testing. 

In general - pre-submit is required to merge, but not sufficient. If a PR is touching areas that we know are not  covered or can't be covered by pre-sumbit - additional testing ( or additional tests ) are needed. For example any perf-related change need to be tested on a load clusters, etc

So please leave it off by default, or move to master.